src/components/Rodape.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const Rodape = () =>{
```

Comment:
Só por padrão, deveríamos ou manter todo o código em ingles, ou em português.
Eu deixo a opção para vocês.
Normalmente eu prefiro inglês simplesmente porque codificar já é em inglês.

Mas, to por vocês @gislainecosta, @ricardomlima, só gostaria de um padrão.

Reply:
Concordo com o padrão em inglês

Reply:
Ok, deixamos em inglês então :)